### DISCUSSION TITLE
[problem solved, please delete this discussion]why there are React and React__default the same time in bundled file? (v10.0.5)

### DISCUSSION THREAD

**Discussion by aqeja** [xyflow/xyflow#2069] [Q&A] [answered]
in the bundle file `nocss/index3.js`

```ts
import * as React from 'react';
import React__default, { memo, useState, useCallback, useEffect } from 'react';
```
react package is imported two times, meanwhile, there are two names for 'react'

however, int some other files, `nocss/index.js` e.g.
```ts
import React__default, { memo } from 'react';
```
react package is imported just once as React__default.
I am confusing, when I'm using bundle tools like vite, the can't import React easily, so am i😭,what should i do?

**Comment by aqeja** [comment, marked as answer]
update to version 10.1.0, problem solved

**Comment by raarts** [comment]
@aqeja Why did you ask to delete this discussion?

**Reply by aqeja** [reply]
version 10.0.5 has this bug, use 10.1.0 or newer version  instead
